src/main/java/core/basesyntax/service/operation/OperationHandler.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+
+public interface OperationHandler {
+    FruitDao fruitDao = new FruitDaoImpl();
```

Comment:
Please avoid hardcoding dependencies with new. It often makes code not testable and not maintainable.
Use constructor in implementations instead